thrift/src/main/thrift/atoms/chart.thrift

```diff
@@ -0,0 +1,83 @@
+namespace * contentatom.chart
+namespace java com.gu.contentatom.thrift.atom.chart
+
+enum ChartType {
+    BAR = 0
+    BARGROUP = 1
+    BARGROUPSTACK = 2
+    BARGROUPSTACK100 = 3
+    BAR100 = 4
+    BROKENBAR100 = 5
+    COL = 6
+    COLGROUP = 7
+    COLGROUPSTACK = 8
+    COLGROUPSTACK100 = 9
+    LINEDISCRETE = 10
+    LINECONTINUE = 11
+    PLOTDOT = 12
+    PLOTSCATTER = 13
+    ONBARDIFFARROW = 14
```

Comment:
🧐 I don't know what that means

Reply:
Same! This is a copy of the available charts in the [current tool](https://github.com/guardian/interactive-basichartool/blob/master/src/components/charts.js#L26-L49).

Reply:
Perhaps we can ask for clarification, I'm worried this may lead to a lot of duplication in the code (the different chart types do seem to overlap a lot). It may be that a `ChartOptions` data type must be created for defining variants of a bar/line/dot chart.

Reply:
@regiskuckaertz , @susiecoleman and I are finding it tricky to reason about all the chart types, so we're going to model `bar` for now to get an end to end process defined. Then additional chart types should be a matter of bumping libraries.

Sound good to you?